src/common-principles.md

```diff
@@ -536,12 +550,39 @@ like in the example below.
             "F": {
                 "Description": "Female",
                 "TermURL": "https://www.ncbi.nlm.nih.gov/mesh/68005260"
-            },
+            }
         }
     }
 }
 ```
 
+### Compressed tabular files
```

Comment:
I would make this a sub-heading under tabular files.

```suggestion
#### Compressed tabular files
```

Reply:
I'm not a fan of this; creating one more level of depth without siblings feels artificial (\*), and it also makes the section less findable. Curious about what others think.

(\*) At least, it would seem logical to do:

```
## Tabular files
### Plain-text tabular files
### Compressed tabular files
```